extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -26,11 +155,21 @@ export let KeyedShoppingConfigDef;
 export function getShoppingConfig(pageElement) {
   const element = pageElement.querySelector('amp-story-shopping-config');
   return getElementConfig(element).then((config) => {
-    //TODO(#36412): Add call to validate config here.
-    return keyByProductTagId(config);
+    config['items'].forEach((itemConfig) => validateConfig(itemConfig));
+
+    if (essentialFieldsAccum.length > 0) {
+      user().warn(
+        'AMP-STORY-SHOPPING-CONFIG',
+        `Essential fields ${JSON.stringify(
+          essentialFieldsAccum
```

Comment:
`essentialFieldsAccum.join[', ']` instead of making it a JSON?

Reply:
Sounds good, I'll be removing this variable anyway as per Corey's suggestion:
https://github.com/ampproject/amphtml/pull/37474#discussion_r806219784